### ISSUE TITLE
Training from scratch

### ISSUE THREAD

**Issue by ntkarim** [AlfredXiangWu/face_verification_experiment#105]
Hello, thank you for your model first of all. I tried training from scratch the lightened CNN on CASIA dataset but I got training loss was 0. Moreover, I tried finetuning the model that was given on another dataset but still I got training loss of 0. I am using non-aligned images for training in both cases. Is there anything I am doing wrong or could you please give directions on how to finetune or train the lightened CNN from scratch?

**Comment by AlfredXiangWu** [comment]
I suggest that you should use aligned face images to train the light CNN. 

Besides, if the loss is 0, I think something is wrong for your configuration. The initial loss should be about 9\~11. It would be gradually decreased and the final loss should be about 0.03\~0.05.

**Comment by ZHAIXINGZHAIYUE** [comment]
it's really 0.03-0.05 not 0.3-0.5? Would you like to share your train log with us?
If the final loss is 0.03_0.05, it means the model will correct classify the the train data with the accuracy 0.95-0.97? Also, the model B and model C all will be converaged with the loss 0.03-0.05?
Thank you.
